src/services/api/actions/getSearchResults.js

```diff
@@ -4,11 +4,12 @@ import { useStateValue } from '../../../store/store';
 export const getSearchResults = ({ unit = 20, keyword, currentPage = 1 }) => {
 	const [{ searchSiteFilter }] = useStateValue();
 	const endpoint = getEndpoint('searchResults');
-	const offSet = currentPage === 1 ? 0 : (currentPage - 1) * unit + 1;
+	currentPage -= 1;
+	currentPage = currentPage < 1 ? 0 : currentPage * unit;
```

Comment:
nit: could this be a better name like, `calculatedFrom` because it isnt a page in this context

Reply:
👍 `resultIndex`, `offset`, or `computedIndex`

Reply:
Also wondering if this might be clearer without the ternary in the computation:
```javascript
const computedIndex = unit * (currentPage - 1);
```
And a check to ensure `currentPage` is never less than 1